File: RAM Size/ram.c
/*
	Minimum number of browsers, C
	Module written by Brandon Arvanaghi
	Website: arvanaghi.com
	Twitter: @arvanaghi
*/

#include <Windows.h>
#include <stdio.h>

int main() {
	MEMORYSTATUSEX memStat;

	memStat.dwLength = sizeof(memStat);
	GlobalMemoryStatusEx(&memStat);

	if ((float)memStat.ullTotalPhys / 1073741824 > 1) {
		printf("The RAM of this host is at least 1 GB in size. Proceed!\n");
	} else {
		printf("Less than 1 GB of RAM exists on this system. Do not proceed.\n");
	}

	getchar();
	return 0;
}

File: RAM Size/ram.pl
#
#   Minimum number of browsers, Perl
#   Module written by Brandon Arvanaghi 
#   Website: arvanaghi.com 
#   Twitter: @arvanaghi
#

use Win32::OLE qw(EVENTS HRESULT in);

my $wmiCIM = Win32::OLE->GetObject("WINMGMTS://./root/CIMv2");

my $RAM_running_total = 0;
foreach my $subMem (in($wmiCIM->InstancesOf('Win32_PhysicalMemory'))) {
    $RAM_running_total += $subMem->{Capacity};
}

if ($RAM_running_total/1073741824 > 1) {
	print "The RAM of this host is at least 1 GB in size. Proceed!\n";
} else {
	print "Less than 1 GB of RAM exists on this system. Do not proceed.\n"
}